mdfreader/mdfinfo4.py

```diff
@@ -2305,7 +2373,10 @@ def _unique_channel_name(self, fid, name, dg, cg, cn):
                     source_name = dg
             else:
                 source_name = dg
-            name = u'{0}_{1}_{2}'.format(name, dg, source_name)
+            if multi_sources and name != 't':
```

Comment:
This is very specific to me. I suppose "t" is time. Can very different system by system, not really portable

Reply:
I've made an adjustment concerning the master channel(time in my acquisitions), aiming for a unique name for that channel. Specifically, I replaced it with "nfo['masters'][info['DG'][dataGroup]['dg_cg_first']]['name']". I would value your insights and thoughts on this modification. If you have any suggestions or considerations regarding this change, please feel free to share.